core/src/main/java/org/wildfly/extras/creaper/core/online/ManagementProtocol.java

```diff
@@ -15,6 +15,15 @@
      * Default port 9990.
      */
     HTTP("http"),
+    /**
+     * With {@code HTTPS}, the management client will not use the native management protocol,
+     * but will instead use the HTTPS management endpoint. The {@code ModelNode}s will be serialized to JSON. Operation
+     * attachments are not supported (yet), so deploying will not work. If username and password is not provided,
+     * no authentication method will be used.
```

Comment:
We should probably just say something like "HTTPS imposes the same limitations as {@link #HTTP}" and not repeat them all.